hack/ensure-golangci-lint.sh

```diff
@@ -18,6 +18,10 @@
 
 set -e
 
+if [[ "${TRACE-0}" == "1" ]]; then
```

Comment:
`x` is also accepted as an arg in this script at `L54`. It might be a good idea to remove that for this script (the x argument isn't documented in usage either, so it would be good to fix that up one way or another)

Reply:
Good catch. Fixed